### DISCUSSION TITLE
Expand the input during reviewing

### DISCUSSION THREAD

**Discussion by HiskeB** [asreview/asreview#1040] [Q&A] [answered]
Hi, 

I am wondering whether is it possible to import extra data (from another database) in my ASReview project even though I've already (just) started the process of reviewing. 

Thanks in advance, 
Hiske

**Comment by FelixWdm** [comment, marked as answer]
Hi @HiskeB , what you can do, is stop reviewing in this project. Export the project, so you'll have all the records, including the included/excluded and combine this with your new dataset. If you use this to start a new project, the model will be trained using all available knowledge and the newly added records will be ranked along with the rest from most > least relevant. Check the [manual ](https://asreview.readthedocs.io/en/latest/features/pre_screening.html#partly-labeled-data)on how to do this. You can most easily add these in the table form, a csv file.
I hope this helps, if not let us know.

**Comment by gimoAI** [comment]
It is now also possible to use [compose](https://github.com/asreview/asreview-datatools#data-compose-experimental) from the [ASReview datatools extension](https://github.com/asreview/asreview-datatools) for this purpose. There is a [tutorial on how to update a systematic review](https://github.com/asreview/asreview-datatools/blob/master/Tutorials.md#1-update-systematic-review) that you are working on in ASReview.
